pyext/src/clustering_rmsd.py

```diff
@@ -23,12 +24,11 @@ def get_sample_identity(idfile_A, idfile_B):
 
 
 def get_cutoffs_list(distmat, gridSize):
-
     mindist = distmat.min()
     maxdist = distmat.max()
 
     print("Minimum and maximum pairwise model distances:", mindist, maxdist)
-    cutoffs = numpy.arange(mindist+gridSize, maxdist, gridSize)
+    cutoffs = numpy.arange(mindist + gridSize, maxdist, gridSize)
```

Comment:
Most of the changes in this PR appear to be minor formatting changes like this one. This makes it quite hard to spot what it is you actually changed. I'm certainly not opposed to making the code easier to read if you really want to, but it should be in a separate PR.